.github/workflows/ci.yml

```diff
@@ -33,18 +33,23 @@ jobs:
           uv venv
           uv pip install -e ".[dev]"
 
+      - name: Run checks
```

Comment:
I will be sad if I wait 3 minutes for tests to run only to realize the code was not formatted